libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -990,6 +990,8 @@ bool ExpressionCompiler::visit(FunctionCall const& _functionCall)
 		}
 		case FunctionType::Kind::Error:
 		{
+			// This case is part of the ``revert Error()`` path of the codegen.
+			// For ``require(bool, Error())`` refer to the ``Kind::Require`` case.
```

Comment:
https://github.com/ethereum/solidity/pull/15174#discussion_r1639971856
```suggestion
			// This case is part of the ``revert <error>`` path of the codegen.
			// For ``require(bool, <error>)`` refer to the ``Kind::Require`` case.
```